src/widgets/forms.js

```diff
@@ -28,8 +32,20 @@ const dashCharacter = '-'
 /*                                  Form Field implementations
 **
 */
-/*          Group of different fields
+/**          Group of different fields
+**
+**  One type of form field is an ordered Group of other fields.
+**  A Form is actually just the same as a group.
+**
+** @param {Document} dom The HTML Document object aka Document Object Model
```

Comment:
Very happy to have more documentation, but a lot of this will be automatically generated when we migrate to TypeScript and use [TypeDoc](https://typedoc.org/).

But lets have this here for now ^_^